Packs/Netmiko/Integrations/Netmiko/Netmiko.py

```diff
@@ -39,7 +45,7 @@ def connect(self):
         else:
             try:
                 self.net_connect = Netmiko(device_type=self.platform, host=self.hostname, port=self.port,
-                                           use_keys=True, username=self.username, password=self.password)
+                                           use_keys=False, username=self.username, password=self.password)
```

Comment:
what's the reason for this change?

Reply:
use_keys was changed to False since we aren't passing an SSH Key Certificate as part of the credentials parameter. As such, we are simply using a standard username/password from the credential parameter.